README.md

```diff
@@ -405,12 +396,12 @@ export async function generateEmbedTokens(userAgent, user) {
 }
 ```
 
-### Initializing the Looker SDK in the frontend
+### Initializing the Embed SDK in the frontend
 
-Cookieless embed is initialized by calling `LookerEmbedSDK.initCookieless` passing in the Looker host value and the the urls of the backend endpoints described previously. Once a Looker embed IFRAME is created it will communicate with the Embed SDK running in the host application and use the callbacks appropriately.
+Cookieless embed is initialized by calling `getEmbedSDK().initCookieless(...)` passing in the Looker host value and the the urls of the backend endpoints described previously. Once a Looker embed IFRAME is created it will communicate with the Embed SDK running in the host application and use the callbacks appropriately.
```

Comment:
```suggestion
Cookieless embed is initialized by calling `getEmbedSDK().initCookieless(...)` passing in the Looker host value and the urls of the backend endpoints described previously. Once a Looker embed IFRAME is created it will communicate with the Embed SDK running in the host application and use the callbacks appropriately.
```